tests/ui/traits/new-solver/higher-ranked-dyn-bounds.stderr

```diff
@@ -0,0 +1,18 @@
+error[E0275]: overflow evaluating the requirement `for<'a> dyn for<'a> Trait<'a, for<'a> Item = ()>: Trait<'a>`
```

Comment:
This goal causes one of those infinite runaways of placeholders in increasingly higher universes, like we see with WF goals currently.